tests/components/homewizard/test_init.py

```diff
@@ -97,85 +97,6 @@ async def test_load_removes_reauth_flow(
     assert len(flows) == 0
 
 
-@pytest.mark.parametrize(
-    "exception",
-    [
-        HomeWizardEnergyException,
-        Exception,
-    ],
-)
-async def test_load_handles_homewizardenergy_exception(
-    hass: HomeAssistant,
-    mock_config_entry: MockConfigEntry,
-    mock_homewizardenergy: MagicMock,
-    exception: Exception,
-) -> None:
-    """Test setup handles exception from API."""
-    mock_homewizardenergy.device.side_effect = exception
-    mock_config_entry.add_to_hass(hass)
-    await hass.config_entries.async_setup(mock_config_entry.entry_id)
-    await hass.async_block_till_done()
-
-    assert mock_config_entry.state in (
-        ConfigEntryState.SETUP_RETRY,
-        ConfigEntryState.SETUP_ERROR,
-    )
-
-
-@pytest.mark.parametrize(
-    ("device_fixture", "old_unique_id", "new_unique_id"),
-    [
-        (
-            "HWE-SKT-11",
-            "aabbccddeeff_total_power_import_t1_kwh",
-            "aabbccddeeff_total_power_import_kwh",
-        ),
-        (
-            "HWE-SKT-11",
-            "aabbccddeeff_total_power_export_t1_kwh",
-            "aabbccddeeff_total_power_export_kwh",
-        ),
-        (
-            "HWE-SKT-21",
-            "aabbccddeeff_total_power_import_t1_kwh",
-            "aabbccddeeff_total_power_import_kwh",
-        ),
-        (
-            "HWE-SKT-21",
-            "aabbccddeeff_total_power_export_t1_kwh",
-            "aabbccddeeff_total_power_export_kwh",
-        ),
-    ],
-)
-@pytest.mark.usefixtures("mock_homewizardenergy")
-async def test_sensor_migration(
-    hass: HomeAssistant,
-    entity_registry: er.EntityRegistry,
-    mock_config_entry: MockConfigEntry,
-    old_unique_id: str,
-    new_unique_id: str,
-) -> None:
-    """Test total power T1 sensors are migrated."""
-    mock_config_entry.add_to_hass(hass)
-
-    entity: er.RegistryEntry = entity_registry.async_get_or_create(
-        domain=Platform.SENSOR,
-        platform=DOMAIN,
-        unique_id=old_unique_id,
-        config_entry=mock_config_entry,
-    )
-
-    assert entity.unique_id == old_unique_id
-
-    assert await hass.config_entries.async_setup(mock_config_entry.entry_id)
-    await hass.async_block_till_done()
-
-    entity_migrated = entity_registry.async_get(entity.entity_id)
-    assert entity_migrated
-    assert entity_migrated.unique_id == new_unique_id
-    assert entity_migrated.previous_unique_id == old_unique_id
-
-
```

Comment:
Not really sure, but I _think_ the test below can be removed as well

Reply:
Done! [ead9aba](https://github.com/home-assistant/core/pull/122086/commits/ead9abad204b81d6160268c91d7fa70e1301f3ea)